ansible/files/multinode.sh

```diff
@@ -365,6 +365,7 @@ function upgrade_overcloud() {
 
   run_kayobe overcloud host upgrade
   run_kayobe overcloud host configure
+  run_kayobe overcloud database recover
```

Comment:
Not 100% opposed to this, but wouldn't it be better to serialize the reboots to prevent this? Do you know what task triggers it? My recollection is that the CIS playbooks won't actually trigger a reboot, but I could imagine it changing a setting like (selinux) which would cause kayobe internal playbooks to reboot them

Reply:
Also, would be happier to approve this as-is if it came with a FIXME like comment so that we can remove it in the future (we shouldn't need to do this long term). That could be the fastest way to "fix" the CI?

Reply:
I've just double-checked and you're right, it was selinux (specifically [this bit](https://github.com/openstack/kayobe/blob/master/ansible/roles/selinux/tasks/main.yml#L39-L43)) It'll take a little while to get a change in there, so I'll go with your FIXME approach for now and update it when I can.

Reply:
Done